core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -552,6 +564,7 @@ private int sendOrProcessProvisionedRequests() {
         return processes.size();
     }
 
+    @WithSpan("process_provider_request")
```

Comment:
"process_transfer_process_request"? - the provider part will be clear from the traces as it will be shown in the provider component